spec/support/page_helpers.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module PageHelpers
+  def reload_page_until_timeout!(text:, as_link: false)
+    Timeout.timeout(Settings.timeouts.workflow) do
+      loop do
+        # NOTE: Using passing `true` to this JavaScript function should force
+        #       the browser to bypass its cache.
+        page.evaluate_script('location.reload(true);')
+
+        # NOTE: This could have been a ternary but I was concerned about its
+        #       readability.
+        if as_link
+          break if page.has_link?(text)
+        else
+          break if page.has_text?(text)
+        end
```

Comment:
```suggestion
        break if (as_link && page.has_link?(text)) || page.has_text?(text)
```

Reply:
@jcoyne I'm open to this but I wonder if it's much better than the ternary. It's essentially a ternary with different syntax, no? @sul-dlss/infrastructure-team ?

Reply:
I find the if / else easier to puzzle out at a glance, but it is only a slight preference.

Reply:
How about `break if condition_satisfied?(text, as_link)`

Reply:
@jcoyne I considered that too, but it felt gross to be passing all that state around. I'm open to that as well. I'm basically open to anything at least two potential approvers can agree on. :laughing:

Reply:
`if / else` is clearer without mental exertion.

Reply:
i would say the code as it stands is an improvement over the pre-PR code, so it has my approval now or with any other way to handle this very tiny style concern.  I don't care - let's get the fix in!

Reply:
Thanks, team. @jcoyne given :point_up:, are you open to changing the 'request changes' on this PR?